code/game/objects/structures/lattice.dm

```diff
@@ -21,7 +21,8 @@
 	..()
 	for(var/obj/structure/lattice/LAT in loc)
 		if(LAT != src)
-			qdel(LAT)
+			qdel(src)
+			return
```

Comment:
uh. A bunch of stuff relies on the new lattice replacing the lattice that currently exists, please account for that.